docs/osindex.html

```diff
@@ -178,6 +179,7 @@
 
         map.getSource('localBuildings').setData(geojson);
         document.getElementById('feature-count').innerHTML = '';
+        document.getElementById('area').innerHTML = 0;
```

Comment:
it shows 0 for a brief.... perhaps leave it empty until when you have the number.

Reply:
Made this change